src/view/com/composer/Composer.tsx

```diff
@@ -682,6 +688,12 @@ export const ComposePost = ({
             />
           </ComposerTopBar>
 
+          {/* (Note) APiligrim
+          Toast container for iOS and Android modal compatibility only */}
+          {isNative && (
+            <ComposerToastContainer toasts={toasts} removeToast={removeToast} />
+          )}
```

Comment:
Interesting solution! Looks like this works fine, but I think all we need to do is wrap the composer in another `RootSiblingParent` and the existing toasts will work as normal. Seems to work fine on iOS and web for me.

So that's just adding `<RootSiblingParent>...</RootSiblingParent>` at line 660 in `composer/Composer.tsx`.

Def reach out if you run into more tricky stuff like this, our codebase is full of little gotchas like this!

Reply:
thank you so much for pointing this out 🙏  I made the updates to reuse the new toast and added the `<RootSiblingParent>` . I tested them on IOS, web, and android again and the toasts show up for me!